common/tvm_utility/example/yolo_v2_tiny/main.cpp

```diff
@@ -239,24 +243,39 @@ class PostProcessorYoloV2Tiny : public tvm_utility::pipeline::PostProcessor<std:
   std::vector<std::pair<float, float>> anchors{};
 };
 
-TEST(PipelineExamples, SimplePipeline)
+}  // namespace yolo_v2_tiny
+}  // namespace tvm_utility
+
+bool check_near(double expected, double actual, double tolerance)
 {
+  return fabs(expected - actual) <= tolerance;
+}
+
+int main(int argc, char * argv[])
+{
+  // init node to use parameters
+  rclcpp::init(argc, argv);
+  auto node = rclcpp::Node::make_shared("tvm_yolo_example");
+  node->declare_parameter("image_filename", IMAGE_FILENAME);
+  node->declare_parameter("label_filename", LABEL_FILENAME);
+  node->declare_parameter("anchor_filename", ANCHOR_FILENAME);
+  node->declare_parameter("data_path", "");
```

Comment:
Parameter declaration should follow https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/ros-nodes/parameters/

- no default value in the code
- a config file with the default values
- a schema file for the template
- you can then make use of auto-generation of parameter table in the documentation (https://github.com/autowarefoundation/autoware.universe/commit/63033beebc9d45b6185b8ec789f41a7b379ac04a#diff-f5557a723bf6eba2f59cb3ac58f670f0ad7ea7f189fa7c868289ffbc2ab574d9)

Reply:
It seems like there is an issue with auto-generation 
![image](https://github.com/autowarefoundation/autoware.universe/assets/37497658/4cbb3bdd-c1e9-4f72-9f71-63ee24b2077f)
https://github.com/autowarefoundation/autoware.universe/tree/main/perception/lidar_apollo_segmentation_tvm_nodes

Reply:
That's expected. It gets rendered here: https://autowarefoundation.github.io/autoware.universe/main/perception/lidar_apollo_segmentation_tvm_nodes/#parameters

Reply:
Does `$(find-pkg-share tvm_utility)` in config/yolo_v2_tiny_example.param.yaml work? I have never seen it used in a config file before.

Reply:
yes, it works with `allow_substs="true"` in launch file. It takes a while to find it for [ros2](https://docs.ros.org/en/foxy/Releases/Release-Galactic-Geochelone.html).

Reply:
that's great!